task/s2i-dotnet/0.1/README.md

```diff
@@ -1,25 +1,28 @@
-# .NET Core 3 Source-to-Image
+# .NET Core Source-to-Image
 
-This task can be used for building `.NET Core 3` apps as reproducible Docker
+This task can be used for building `.NET Core` apps as reproducible Docker
 images using Source-to-Image. [Source-to-Image (S2I)](https://github.com/openshift/source-to-image)
 is a toolkit and a workflow for building reproducible container images
 from source code. This tasks uses the s2i-dotnet image build from [redhat-developer/s2i-dotnetcore](https://github.com/redhat-developer/s2i-dotnetcore).
 
 .NET Core versions currently provided are:
+- 2.1
+- 2.7-el7  
+- 2.1-ubi8
```

Comment:
Since we didn't support version 2.1 in previous releases, do we want to introduce the support now? @vdemeester @siamaksade

Reply:
If we didn't before, I would note, except if there is a users/customers asking for it.